src/foundry/client/pixi/layers/placeables/drawings.d.ts

```diff
@@ -0,0 +1,123 @@
+export {};
+
+declare global {
+  /**
+   * The DrawingsLayer subclass of PlaceablesLayer.
+   */
+  class DrawingsLayer extends PlaceablesLayer<"Drawing"> {
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    static get instance(): Canvas["drawings"];
+
+    /**
+     * @remarks This is not overridden in foundry but reflects the real behavior.
+     */
+    override options: DrawingsLayer.LayerOptions;
+
+    /**
+     * @defaultValue
+     * ```
+     * mergeObject(super.layerOptions, {
+     *   name: "drawings"
+     *   canDragCreate: true,
+     *   controllableObjects: true,
+     *   rotatableObjects: true,
+     *   zIndex: 20
+     * })
+     * ```
+     */
+    static override get layerOptions(): DrawingsLayer.LayerOptions;
+
+    static override documentName: "Drawing";
+
+    /**
+     * The named game setting which persists default drawing configuration for the User
+     */
+    static DEFAULT_CONFIG_SETTING: "defaultDrawingConfig";
+
+    /**
+     * Use an adaptive precision depending on the size of the grid
+     */
+    get gridPrecision(): 0 | 8 | 16;
+
+    override get hud(): Exclude<Canvas["hud"], undefined>["drawing"];
+
+    override get hookName(): (typeof DrawingsLayer)["name"];
+
+    /**
+     * Render a configuration sheet to configure the default Drawing settings
+     */
+    configureDefault(): void;
+
+    override _deactivate(): this;
+
+    /**
+     * Get initial data for a new drawing.
+     * Start with some global defaults, apply user default config, then apply mandatory overrides per tool.
+     * @param origin - The initial coordinate
+     * @returns The new drawing data
+     * @remarks This is used from DrawingConfig and hence public on purpose.
+     */
+    _getNewDrawingData(origin: Point | {}): NewDrawingData;
```

Comment:
The `{}` type basically is `Exclude<any, undefined | null>`. That means you can pass an array, a number, a string, etc. to this function. The actual type that resolves to is `any` but the point is that you can assign basically anything but `undefined` and `null` to `{}`. The type you want for an _object_ with no required properties you want `Record<string, unknown>`. If you want an object that NEVER has any properties you want `Record<string, never>`. The difference is a parameter like `{ foo: 1 }` is valid if the type is `Record<string, unknown>` but not valid if the type is `Record<string, never>`